### PR TITLE
Adds a batch of OLM and RHCOS RNs

### PR DESCRIPTION
**Pull request by stevsmit**
Preview: https://deploy-preview-37060--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-bug-fixes

For 4.9. 

Can be merged upon approval.

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -1024,6 +1024,10 @@ The deprecated `dhclient` binary has been removed from {op-system}. Starting wit
 
 *Operator Lifecycle Manager (OLM)*
 
+* Previously, Operator Lifecycle Manager upgradeable condition message was unclear when installed Operators set `olm.maxOpenShiftVersion` to a minor {product-title} (OCP) version less than or equal to the current OCP version. This resulted in an incorrect error message that has been fixed to specify that only minor and major version upgrades are blocked when `olm.maxOpenShiftVersion` is set to version different than the current OCP version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1992677[*BZ#1992677*])
+
+* Previously, the `opm` command failed to deprecate bundles when they were present in the index. Consequently, bundles truncated as part of another deprecation in the same call were reported as missing. This update adds a check for bundles before any deprecation takes place to differentiate between a bundle that is not present and one that has been truncated. As a result, deprecating bundles along the same upgrade path no longer report as missing. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1950534[*BZ#1950534*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-9-release-notes.adoc:1029]:
```suggestion
* Previously, the `opm` command failed to deprecate bundles when they were present in the index. Consequently, bundles truncated as part of another deprecation in the same call were reported as missing. This update adds a check for bundles before any deprecation takes place to differentiate between a bundle that is not present and one that has been truncated. As a result, deprecated bundles along the same upgrade path are no longer reported as missing. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1950534[*BZ#1950534*])
```

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -1097,6 +1101,8 @@ conditions:
 
 * In an s390x Kernel Virtual Machine (KVM) with an ECKD DASD attached, the DASD would appear to be a regular virtio storage device but would become inaccessible if the VTOC was removed. As a result, you could not use DASD as a virtio block device when installing {op-system-first} on the KVM. The `coreos-installer` program has been updated so that it now installs RHCOS with a VTOC-format partition table when the installation destination is a virtio storage device such as an ECKD DASD attached to a KVM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1960485[*BZ#1960485*])
 
+* Previously, the `NetworkManager-wait-online-service` timed out too early, which prevented a connection to be established before the `coreos-installer` program could start. Consequently,  the `coreos-installer` program failed to fetch the Ignition config if the network took too long to start. With this update, the `NetworkManager-wait-online-service` time out has been increased to its default upstream value. As a result, the `coreos-installer` program no longer fails to fetch the Ignition config. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1967483[*BZ#1967483*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-9-release-notes.adoc:1104]:
```suggestion
* Previously, the `NetworkManager-wait-online-service` timed out too early, which prevented establishing a connection before the `coreos-installer` program could start. Consequently, if the network took too long to start, the `coreos-installer` program failed to fetch the Ignition config. With this update, the `NetworkManager-wait-online-service` timeout has been increased to its default upstream value. As a result, the `coreos-installer` program no longer fails to fetch the Ignition config. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1967483[*BZ#1967483*])
```

**Review by jeana-redhat — COMMENTED** [review]
Couple little things, and needs rebase with the fancy new headers :)

File: release_notes/ocp-4-9-release-notes.adoc [hunk 3]
```diff
@@ -1024,6 +1024,10 @@ The deprecated `dhclient` binary has been removed from {op-system}. Starting wit
 
 *Operator Lifecycle Manager (OLM)*
 
+* Previously, Operator Lifecycle Manager upgradeable condition message was unclear when installed Operators set `olm.maxOpenShiftVersion` to a minor {product-title} (OCP) version less than or equal to the current OCP version. This resulted in an incorrect error message that has been fixed to specify that only minor and major version upgrades are blocked when `olm.maxOpenShiftVersion` is set to version different than the current OCP version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1992677[*BZ#1992677*])
```
[the remark this answers is not in the document]
Comment (stevsmit) [reply release_notes/ocp-4-9-release-notes.adoc:1027]: I think I found that in 4.8... :D

**Comment by jeana-redhat** [comment]
For https://issues.redhat.com/browse/OSDOCS-2514

**Pull request merged**